.github/workflows/build_and_publish.yaml

```diff
@@ -0,0 +1,48 @@
+name: Build and upload to PyPI
+on: [push]
+
+jobs:
+  build_sdist:
+    name: Build source distribution
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          python-version: '3.8'
+          miniforge-variant: Mambaforge
+          miniforge-version: 4.11.0-2
+          use-mamba: true
+          environment-file: environment.yml
+          activate-environment: datajudge
+
+      - name: Build
+        shell: bash -l {0}
+        # It seems as though flit publish expects a .pypirc file as compared to
+        # only a token passed as an argment.
+        # See https://flit.pypa.io/en/latest/cmdline.html#flit-publish
+        run: |
+          flit build --format sdist
```

Comment:
Any particular reason why building an sdist and not a wheel?

Reply:
Building both now, good catch